lua/treesitter-context/render.lua

```diff
@@ -60,21 +82,21 @@ local function display_window(bufnr, winid, width, height, col, ty, hl)
       zindex = config.zindex,
       border = sep and { '', '', '', '', sep, sep, sep, '' } or nil,
     })
-    vim.w[winid][ty] = true
-    vim.wo[winid].wrap = false
-    vim.wo[winid].foldenable = false
-    vim.wo[winid].winhl = 'NormalFloat:' .. hl
+    vim.w[float_winid][ty] = true
+    vim.wo[float_winid].wrap = false
+    vim.wo[float_winid].foldenable = false
+    vim.wo[float_winid].winhl = 'NormalFloat:' .. hl
   else
-    api.nvim_win_set_config(winid, {
-      win = api.nvim_get_current_win(),
+    api.nvim_win_set_config(float_winid, {
+      win = winid,
       relative = 'win',
       width = width,
       height = height,
       row = 0,
       col = col,
     })
   end
-  return winid
+  return float_winid
 end
 
 --- @param winid integer
```

Comment:
textoff is inconsistent too. In my case it will return 4 and after next update time it will return 8, cause window position to flash